model/openapi.yml

```diff
@@ -283,6 +283,10 @@ components:
           description: The artifact download progress as a percentage.
           type: number
           format: double
+        download_rate:
```

Comment:
We also added two more download fields `retry_error_msg` and `retry_until`, let's add those while we are in here:

They came from https://github.com/elastic/elastic-agent/pull/3845

Reply:
Relates https://github.com/elastic/kibana/issues/173370

Reply:
Added here: https://github.com/elastic/fleet-server/pull/3177/commits/e0a0188a5f90e979d1f659e28bd64e28cd952002